F/FastTransforms/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+using BinaryBuilder
+
+# Collection of sources required to build FastTransforms
+name = "FastTransforms"
+version = v"0.2.9"
+sources = [
+    "https://github.com/MikaelSlevinsky/FastTransforms/archive/v$(version).tar.gz" =>
+    "555e8fb19ad76ee888fbab7bfaf4c269416c1c5950e9428cbe08edb44c22bf35",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+if [[ ${target} != *darwin* ]]; then
+    if [[ ${target} == *mingw* ]]; then
+        cd $WORKSPACE/destdir/bin
+    else
+        cd $WORKSPACE/destdir/lib
+    fi
+    if [[ ${nbits} == 32 ]]; then
+        ln -sf libopenblas.${dlext} libblas.${dlext}
+    else
+        ln -sf libopenblas64_.${dlext} libblas.${dlext}
+    fi
+fi
+
+cd $WORKSPACE/srcdir/FastTransforms-*
+
+if [[ ${target} == *mingw* ]]; then
+    gcc  -std=gnu99 -Ofast -march=native -mtune=native -mno-vzeroupper -I./src -I/workspace/destdir/include -lm -shared -fPIC src/transforms.c src/rotations.c src/permute.c src/tdc.c src/drivers.c src/fftw.c -L/workspace/destdir/bin -lm -lquadmath -fopenmp -lblas -lfftw3 -lgmp -lmpfr -o libfasttransforms.dll
+    cp -a libfasttransforms.${dlext} ${prefix}/bin
+else
+    make lib CC=gcc FT_USE_PREDEFINED_LIBRARIES=1 FT_FFTW_WITH_COMBINED_THREADS=1
+    cp -a libfasttransforms.${dlext} ${prefix}/lib
+fi
+"""
+
+platforms = expand_gfortran_versions([Linux(:i686, libc=:glibc);
```

Comment:
Any particular reason for not using all `supported_platforms()`?

Reply:
The C code uses Intel intrinsics for SIMD, and I’d have to take a closer look at how to support the remaining Linux platforms. Personally, I hope this PR is acceptable once the MacOS and Windows(:x86_64) issues are resolved.

Reply:
The supported platforms are up to you of course, I was just wondering why you're excluding some of them.

You may have noticed that CI on this repository is run separately for each platform, so that you can easily find out which is working and which is not, without a failing platform blocking the rest of the list as it was with the old system.

Reply:
Good to know, thanks!